src/app/schedule-section.tsx

```diff
@@ -11,12 +11,12 @@ const schedule: EventItemProps[] = [
     {
         title: "Opening Ceremony",
         time: "10:00 AM",
-        duration: "20 min",
+        duration: "15 min",
         description: "Check in starting at 9:00 AM"
     },
     {
         title: "Networking",
-        time: "10:20 AM",
+        time: "10:15 AM",
```

Comment:
This is wrong - check the individual view; there's still a five minute buffer so it starts at 10:20